### DISCUSSION TITLE
WPA2 Enterprise where we at ??

### DISCUSSION THREAD

**Discussion by SL2021-Dev** [micropython/micropython#12143] [Core Development]
This is a much need function for commerical micropython projects.
This has been requested since... capitek Thu May 20, 2021 3:49 am
https://forum.micropython.org/viewtopic.php?f=18&t=10543&start=20

I am in the process of updating from 1.17 to hopefully 1.20+ and idf 5.1
Security is becoming more demanded.
I hope someone in the micropython community with the expertise could help !!!

Anyone looking to take their projects into the commerical world will be faced with this need.

**Comment by SL2021-Dev** [comment]
4 more months and no responce.

**Comment by andrewleech** [comment]
This sort of feature would be hard for anyone to work on if they've never used it before and don't have access to an enterprise setting with it running.

I've worked for a few companies myself, often in DevOps style roles, but never encountered it before.

Like lots of domain specific features, this is probably going to need to wait until someone in the community with access to a suitable hardware environment can write the code and submit it.

**Comment by jouellnyc** [comment]
This is not what you want @SL2021-Dev  but here's an infrastructure workaround -- if you will - https://github.com/jouellnyc/wpa-ent-mschapv2-bridge

For others getting here from  google, perhaps it can help -- - I'm guessing others have done this but have not run into many blogs/posts on such a thing.
